framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.8 under development
 -----------------------
 
+- Bug #11270: Error when link record in one to many relation (iushev)
```

Comment:
I'd suggest another wording:

<code>Fixed \`BaseActiveRecord::link()\` method in order to support closure in \`indexBy\` for relations declaration (iushev)</code>